.github/workflows/update-component-versions.yaml

```diff
@@ -60,7 +60,7 @@ jobs:
         shell: bash
         run: |
           sed -i "s/${OLDNUM}/${NEWNUM}/g" docs/deployment/cloud.rst
-          sed -i "s/${OLDNUM}-py310/${NEWNUM}-py310/g" docs/deployment/deploying_custom_image_function.rst
```

Comment:
@akihikokuroda @psschwei I don't know if you can double check the change that I'm doing here. Is this correct?

Reply:
it still keeps the py310 suffix

Reply:
for this one only, you'd need to either:
- use `sed -i "s/0.14.2-py310/0.15.0/g" docs/deployment/deploying_custom_image_function.rst`, or
- just hard delete the suffix from the file

with the former, we'd need to drop the suffix from the "find" part before the next release after 0.15.0